Dockerfile

```diff
@@ -9,20 +9,33 @@ LABEL org.label-schema.vendor="gyptazy"
 LABEL org.label-schema.url="https://proxlb.de"
 LABEL org.label-schema.vcs-url="https://github.com/gyptazy/ProxLB"
 
-# Install Python3
-RUN apk add --no-cache python3 py3-pip
+# --- Step 1 (root): system deps, user, dirs ---
+RUN apk add --no-cache python3 py3-pip py3-virtualenv \
+  && addgroup -S appgroup \
+  && adduser -S -G appgroup -h /home/appuser appuser \
+  && mkdir -p /app/conf /opt/venv \
+  && chown -R appuser:appgroup /app /home/appuser /opt/venv
 
-# Create a directory for the app
 WORKDIR /app
 
-# Copy the python program from the current directory to /app
-COPY proxlb /app/proxlb
+# Copy only requirements first for better layer caching
+COPY --chown=appuser:appgroup requirements.txt /app/requirements.txt
 
-# Copy requirements to the container
-COPY requirements.txt /app/requirements.txt
+# --- Step 2 (appuser): venv + deps + code ---
+USER appuser
```

Comment:
Can we maybe switch the user to `plb` to align it with the Debian and RPM packages where we also use the `plb` user for systemd (see also: https://github.com/gyptazy/ProxLB/blob/main/service/proxlb.service#L8).